PixelHunter1995/PixelHunter1995.csproj

```diff
@@ -53,11 +54,17 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Content\Images\Background1.jpg" />
+    <Content Include="Content\Sounds\495674__jack-urbanski__vibraphone-chord.wav" />
+    <Content Include="Content\Sounds\92734__tj-mothy__slow-sad-tones.wav" />
```

Comment:
Why are these here now when they weren't before?

Reply:
I added some stuff to the solution explorer in Visual Studio. Stuff that exist in the already present `Content` folder, and which I thought was nice to be able to see there.

This doesn't change anything else but the view in Visual Studio. However, further down in this file, you can see that I made the scene XML file be copied to the build directory when building, so it can be read during run-time.